blocks/table/table.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Table Block
+ * Recreate a table
+ * https://www.hlx.live/developer/block-collection/table
+ */
+
+function buildCell(rowIndex) {
+  const cell = rowIndex ? document.createElement('td') : document.createElement('th');
+  if (!rowIndex) cell.setAttribute('scope', 'col');
+  return cell;
+}
+
+export default async function decorate(block) {
+  const table = document.createElement('table');
+  const thead = document.createElement('thead');
+  const tbody = document.createElement('tbody');
+  table.append(thead, tbody);
```

Comment:
find the row with maximum number of cols. If it is fixed (i.e. 3 or 4 cols you need not do that)
`colspan = Math.max(...[...block.children].map(child => child.children.length))`